### DISCUSSION TITLE
winui3  Icon

### DISCUSSION THREAD

**Discussion by bizehao** [microsoft/WinUI-Gallery#886] [Q&A] [answered]
Some icons cannot be displayed, it Into word
![winui3](https://user-images.githubusercontent.com/33247680/159170793-9916891d-80e2-4451-b9b3-6443b413527a.png)
![winui3_question](https://user-images.githubusercontent.com/33247680/159170802-1e54f380-3ae4-497a-8bed-690c351b790a.png)

**Comment by bizehao** [comment, marked as answer]
I set the computer code to utf8, and then it works normally.My previous computer code was GBK

**Reply by gregwoo-microsoft** [reply]
Thank you for listing your solution @bizehao.

**Comment by Jay-o-Way** [comment]
I know why this happend: deprecated icon pointers in the code. niels9001 knows more about that. This shouldn't be a thing anymore, now, is it? See #1238
